packages/iframe-stamper/src/index.ts

```diff
@@ -103,6 +103,16 @@ export type TIframeSettings = {
   styles?: TIframeStyles;
 };
 
+interface PendingRequest {
+  resolve: (value: any) => void;
+  reject: (reason?: any) => void;
+  requestId: string;
+}
+
+function generateUUID(): string {
+  return crypto.randomUUID();
```

Comment:
The iframe stamper will be used strictly in the web browser context, so `crypto` should be an available resource. `randomUUID` is pretty readily available for most modern web browsers.

If we're worried about compatibility, there's an implementation that relies on the older `getRandomValues` (see https://stackoverflow.com/a/2117523)